### PR TITLE
[Docs] Provide example of android side implementation of HeadlessJS

### PR DESCRIPTION
**Pull request by pfulop**
Thanks for submitting a PR! Please read these instructions carefully:

- [x] Explain the **motivation** for making this change.
- [ ] Provide a **test plan** demonstrating that the code is solid.
- [ ] Match the **code formatting** of the rest of the codebase.
- [ ] Target the `master` branch, NOT a "stable" branch.

## Motivation (required)

The existing documentation on this topic doesn't provide enough of detail on where to start a headlessJS task.

## Test Plan (required)

Just docs change

This PR extends HeadlessJS documentation to show an example on how to start HeadlessJsTaskService. Sample code in example reacts to connectivity change using custom broadcast receiver logic that bundles additional information and starts the aforementioned service.

### CODE REVIEW

**Comment by pfulop** [comment]
CLA signed

File: docs/HeadlessJSAndroid.md [hunk 1]
```diff
@@ -57,4 +57,76 @@ Now, whenever you [start your service][0], e.g. as a periodic task or in respons
...
+        /**
+          This part will be called everytime network connection is changed
+          e.g. Connected -> Not Connected
+        **/
+        if (!isAppOnForeground((context))) {
+            /**
+              We will start our service and send extra info about 
+              network connections
+            **/
+            boolean hasInternet = checkInternet(context);
```
Comment (henrikra) [inline docs/HeadlessJSAndroid.md]: You can just use `isNetworkAvailable(context)` here and remove `checkInternet` method totally :)

Comment (pfulop) [inline docs/HeadlessJSAndroid.md]: Yes you are right. It's just some old leftover code I forgot to change. (originally the function did a bit more)

**Comment by hramos** [comment]
Can you update your PR description with more details? E.g. what example are you adding, what does it do? It would also help to add this to the doc itself.

**Comment by hramos** [comment]
Hi there! This issue is being closed because it has been inactive for a while.

But don't worry, it will live on with Canny! Check out its new home: https://react-native.canny.io/feature-requests/p/docs-provide-example-of-android-side-implementation-of-headlessjs

**Pull request closed without merging**